.travis.yml

```diff
@@ -3,10 +3,16 @@ sudo: false
 matrix:
   include:
     - os: linux
-      julia: 0.4
+      julia:
+        - 0.4
+        - 0.5
+        - nightly
```

Comment:
I don't think it likes things being nested. Probably better to just set TESTCMD based on the output of uname